__tests__/app.test.js

```diff
@@ -37,4 +37,43 @@ describe("GET /api/topics", () => {
           });
         });
     });
-})
\ No newline at end of file
+})
+
+describe("GET /api/articles/:article_id", () => {
+    test("200 sends article object that matches the parametric article_ID to the client", () => {
+        const article_id = 5
+        return request(app)
+        .get(`/api/articles/${article_id}`)
+        .expect(200)
+        .then(({ body }) => {
+          expect(typeof body.article.title).toBe('string');
+          expect(typeof body.article.topic).toBe('string');
+          expect(typeof body.article.author).toBe('string');
+          expect(typeof body.article.body).toBe('string');
+          expect(typeof body.article.votes).toBe('number');
+          expect(typeof body.article.created_at).toBe('string');
+          expect(typeof body.article.article_img_url).toBe('string');
```

Comment:
Because we know which article we are asking for, we can test the exact object that will come back (rather than checking the typeof).

Also, there's no need to use a template literal for the url. You can put the 5 straight in there :)